.gitignore

```diff
@@ -1 +1,27 @@
-config.yaml
+# Vim/Nvim Session file
+Session.vim
+
+# Built binaries
+bin/
+
+# If you prefer the allow list template instead of the deny list, see community template:
+# https://github.com/github/gitignore/blob/main/community/Golang/Go.AllowList.gitignore
+#
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+
+# Go workspace file
+go.work
```

Comment:
If you agree with me, we can simplify this file to just 2 or 3 lines, just ignoring the files that this project is generating.